packages/connectivity/src/http-agent/http-agent.ts

```diff
@@ -107,10 +134,14 @@ function getCertificateOption(destination: Destination): Record<string, any> {
     );
   }
 
-  const options = trustAllOptions(destination)({});
+  let options = trustAllOptions(destination)({});
```

Comment:
I would honestly find this way more intuitive, if we would write it out here. Also, I am never sure if I understand `assoc` correctly:
```
let options = { rejectUnauthorized: !destination.isTrustingAllCertificates };
```

(This could also be done in the `trustAllOptions` function).